net-dns/dnscrypt-proxy/dnscrypt-proxy-2.0.0.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGO_PN="github.com/jedisct1/dnscrypt-proxy/..."
+EGO_SRC="github.com/jedisct1/dnscrypt-proxy"
+
+inherit systemd user golang-build
+
+DESCRIPTION="A flexible DNS proxy, with support for modern encrypted DNS protocols."
+HOMEPAGE="https://${EGO_PN}"
+SRC_URI="https://${EGO_PN}/archive/2.0.0.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="systemd"
+
+RDEPEND="systemd? ( sys-apps/systemd )"
+
+S="${WORKDIR}/${P}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.0.0_fix_systemd_service.patch"
+)
+
+pkg_setup() {
+	enewgroup dnscrypt
+	enewuser dnscrypt -1 -1 /var/empty dnscrypt
+}
+
+src_prepare() {
+	default
+	cd "${PN}"
+	GOPATH="${WORKDIR}/${P}:$(get_golibdir_gopath)" go get -u -f "${EGO_PN}"
```

Comment:
Why do you run ````go get```` in a ebuild? It's wrong and it will blow up with FEATURES="network-sandbox"

Reply:
you are right. First time with go lang. I know, I'm a dinosaur. I will remove.